### DISCUSSION TITLE
Egress DNS Control in LinkerD

### DISCUSSION THREAD

**Discussion by wrridgwa** [linkerd/linkerd2#9404] [Q&A] [answered]
Hey, I saw that Linkerd is planning to have egress control on their roadmap. https://linkerd.io/2021/12/29/the-service-mesh-in-2022/#client-side-policy

Do you know if this will allow specifying external DNS endpoints that are whitelisted and preventing all other egress traffic? 

Calico allows specifying IP addresses but this isn't very helpful for our use cases.

Also, any idea when this would become generally available?

**Comment by adleong** [comment, marked as answer]
Hi @wrridgwa.  This kind of egress control is indeed on our roadmap but the work has not yet been properly scoped so I can't really give an estimate.  I suspect we will be laying the groundwork for this in stable-2.13

**Reply by wrridgwa** [reply]
Thank you for the response. Looking forward to this feature!
